### ISSUE TITLE
add ability to not have "space" in `DraxList` when hovered item is not over list

### ISSUE THREAD

**Issue by chrisdrackett** [nuclearpasta/react-native-drax#39]
Currently if you drag an item outside of a `DraxList` the empty space shifts back to the spot that the item came from. This leads to some undesirable visual issues with items jumping around. I would like to make the following suggestion:

1. when an item is dragged outside of its parent list don't show any empty space in that list anymore
2. if the item is released on another receiver everything is good to go
3. if the item is released outside of any receiver, while the item is animating back to its place in the list open the space back up.

Basically the space would open if the item comes back over the list based on the items location OR if the item is released and needs to go back to its original location.

**Comment by lafiosca** [comment]
I think that's a very reasonable behavior and enhancement request. I will try to work on it tonight and see how it goes.

**Comment by lafiosca** [comment]
Ran long working on other issues tonight, hope to take a look at this this weekend. Sorry for the delay.
